code/datums/components/crafting/ranged_weapon.dm

```diff
@@ -328,3 +328,32 @@
 	tool_paths = list(/obj/item/clothing/head/cowboy, /obj/item/clothing/shoes/cowboy)
 	time = 30 SECONDS //contemplate for a bit
 	category = CAT_WEAPON_RANGED
+
+/datum/crafting_recipe/smoothbore_disabler
+	name = "Smoothbore Disabler"
+	result = /obj/item/gun/energy/disabler/smoothbore
+	reqs = list(
+		/obj/item/weaponcrafting/stock = 1, //it becomes the grip
+		/obj/item/stack/cable_coil = 5,
+		/obj/item/pipe = 1,
+		/obj/item/stock_parts/micro_laser = 1,
+		/obj/item/stock_parts/cell = 1,
+		/obj/item/assembly/mousetrap = 1,
+	)
+	tool_behaviors = list(TOOL_SCREWDRIVER, TOOL_WRENCH)
+	time = 10 SECONDS
+	category = CAT_WEAPON_RANGED
+
+/datum/crafting_recipe/smoothbore_disabler_prime
+	name = "Smoothbore Disabler Prime"
```

Comment:
```suggestion
	name = "Heroic Smoothbore Disabler"
```

Just to match with the paired laser musket prime.

Reply:
It uses the Prime name because it's the more fitting one, where the Heroic laser musket is just a better laser musket and has no Prime theming.

If "Prime" is considered too much by the maintainers for the name of the gun, "Perfected", "Artifact" or "Masterwork" would likely be what i'd go with. It's the _pinnacle_ in meth gun technology.

Reply:
I personally avoided 'Prime' because of it being a bit on the nose, and I didn't want to make the reference too direct. I use the language in the code so it helps keep it consistent for mechanical reasons. But 'Masterwork' sounds good too. 'Regal' is also entirely fine if not preferable.

Reply:
It has now become elite.